step-capstone/src/components/TimeLine.js

```diff
@@ -178,13 +177,13 @@ export default function TimeLine(props) {
             div = ":30";
```

Comment:
confused how this works? shouldn't the div be :00 if less than 30? and where does div get set to :00 currently?

Reply:
oh nevermind git was randomly hiding those lines for me very strange